state/charm.go

```diff
@@ -114,13 +119,14 @@ func insertCharmOps(mb modelBackend, info CharmInfo) ([]txn.Op, error) {
 		Config:       safeConfig(info.Charm),
 		Metrics:      info.Charm.Metrics(),
 		Actions:      info.Charm.Actions(),
+		LXDProfile:   safeLXDProfile(lxdProfile),
```

Comment:
same as above.

Reply:
This is safe, because `lxdProfile` will be `nil` and `safeLXDProfile()` returns `nil` if `lxdProfile` is nil. This should be a no-op.

Reply:
I was more thinking because it's a duplicate of the code at line 120.  :-)